src/WebJobs.Extensions.EasyTables/Config/EasyTableHostConfigurationExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Azure.WebJobs.Extensions.EasyTables;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Extension methods for EasyTable integration.
+    /// </summary>
+    public static class EasyTablesHostConfigurationExtensions
+    {
+        /// <summary>
+        /// Enables use of the EasyTable extensions.
+        /// </summary>
+        /// <param name="config">The <see cref="JobHostConfiguration"/> to configure.</param>
+        /// <param name="easyTablesConfig">The <see cref="EasyTableConfiguration"/> to use.</param>
+        public static void UseEasyTables(this JobHostConfiguration config, EasyTableConfiguration easyTablesConfig = null)
```

Comment:
Good suggestion. I think the big one I was missing was EasyTable**s**Configuration. I've changed that now and I think it makes sense.